stable/elasticsearch/templates/client-svc.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
   name: {{ template "elasticsearch.client.fullname" . }}
 {{- if .Values.client.serviceAnnotations }}
   annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
```

Comment:
How does alloying unready pods into the service fix the issue?

Reply:
The pods are unready since they can’t find the service, and the service fails because it doesn’t find any pods. It’s like a circular loop

Reply:
That doesn't answer the question.
We should fix the health check instead of ignore it.